packages/network-controller/src/NetworkController.test.ts

```diff
@@ -909,14 +507,16 @@ describe('NetworkController', () => {
 
   describe('lookupNetwork', () => {
```

Comment:
Since `lookupNetwork` now takes a chain ID, and since this method sets `currentNetworkType` using `getIsCustomNetwork`, and since `getIsCustomNetwork` looks for the chain ID to either be one of 4 chains or not, what do you think about calling this method with different chain ids? The set of possible chain ids, I think, would be similar to [what we are doing for the `providerConfig` setter](https://github.com/MetaMask/core/blob/replace-network-and-iscustomnetwork/packages/network-controller/src/NetworkController.test.ts#L116). So I would expect this to look like:

``` typescript
describe('lookupNetwork', () => {
  describe('if a provider has not been set', () => {
    // ... insert tests that exist now ...
  });

  describe('if a provider has been set, but the resulting EthQuery object does not have a sendAsync method', () => {
    // ... insert tests that exist now ...
  });

  describe('if a provider has been set and the resulting EthQuery object has a sendAsync method', () => {
    ['1', '5', '11155111', ''].forEach((chainId) => {
      describe('if given a chain id of "${chainId}", () => {
        // ... insert tests that exist now, making sure to double check that networkType is set to "infura" ...
      });
    });

    describe('if given a chain id that is not 1, 5, 11155111, or an empty string', () => {
      // ... insert tests that exist now, making sure to double check that networkType is set to "custom" ...
    });
  });
});
```